src/ledger/LedgerManagerImpl.cpp

```diff
@@ -648,7 +648,10 @@ LedgerManagerImpl::historyCaughtup(asio::error_code const& ec,
     if (ec)
     {
         CLOG(ERROR, "Ledger") << "Error catching up: " << ec.message();
-        CLOG(ERROR, "Ledger") << "Catchup will restart at next close.";
+        if (ec == std::make_error_code(std::errc::timed_out))
```

Comment:
This change doesn't make sense to me as it has more to do how `catchup` was invoked:
* this error message should always be ignored when running command line
* this error message should always be displayed when running normally (well it's more complicated than this, but we don't care in the context of this PR)